### ISSUE TITLE
Request to add Graph Tool to Package Manager

### ISSUE THREAD

**Issue by hiroki-o** [Unity-Technologies/AssetGraph#122]
**[Original report](https://bitbucket.org/Unity-Technologies/assetbundlegraphtool/issue/87) by CS (Bitbucket: [optimisez](https://bitbucket.org/optimisez), GitHub: [optimisez](https://github.com/optimisez)).**

----------------------------------------

Starting from Unity 2018, Package Manager has been introduced. It will be really nice if I can always update this awesome tool easily with Package Manager.

**Comment by hiroki-o** [comment]
**Original comment by plunntic iam (Bitbucket: [plunntic](https://bitbucket.org/plunntic), GitHub: [plunntic](https://github.com/plunntic)).**

----------------------------------------

Same here, just wanted to start using it but nope. Rather stick with the asset bundle browser for the time being as it is in the package manager already.

Anyway i'm going to keep an eye on this project, as it looks really powerfull and great for what i will need in a forseable future.

TLDR: +1

**Comment by hiroki-o** [comment]
**Original comment by Hiroki Omae (Bitbucket: [Hiroki Omae](https://bitbucket.org/Hiroki Omae), ).**

----------------------------------------

I have started working on Package Manager support, aiming version 1.5.0 to be delivered through Package Manager.

**Comment by hiroki-o** [comment]
**Original comment by Hiroki Omae (Bitbucket: [Hiroki Omae](https://bitbucket.org/Hiroki Omae), ).**

----------------------------------------

From version 1.5 Asset Graph is installed via package manager. It should show up as preview package in registry later, but meanwhile you can add the package via git URL \(or local package\)
